src/main/java/io/javalin/Javalin.java

```diff
@@ -123,67 +153,134 @@ public Javalin stop() {
         return this;
     }
 
+    /**
+     * Treat '/test/' and '/test' as different URLs.
+     *
+     * The method must be called before {@link Javalin#start()}.
+     */
     public Javalin dontIgnoreTrailingSlashes() {
         ensureActionIsPerformedBeforeServerStart("Telling Javalin to not ignore slashes");
         pathMatcher.setIgnoreTrailingSlashes(false);
         return this;
     }
 
+    /**
+     * Sets custom server implementation.
+     *
+     * @see <a href="https://javalin.io/documentation#custom-server">Documentation example</a>
+     *
+     * The method must be called before {@link Javalin#start()}.
+     */
     public Javalin embeddedServer(@NotNull EmbeddedServerFactory embeddedServerFactory) {
         ensureActionIsPerformedBeforeServerStart("Setting a custom server");
         this.embeddedServerFactory = embeddedServerFactory;
         return this;
     }
 
+    /**
+     * Serves static files from path in classpath.
+     *
+     * The method must be called before {@link Javalin#start()}.
+     *
+     * @see <a href="https://javalin.io/documentation#static-files>Static files in docs</a>
+     */
     public Javalin enableStaticFiles(@NotNull String classpathPath) {
         return enableStaticFiles(classpathPath, Location.CLASSPATH);
     }
 
+    /**
+     * Serves static files from path in given location.
+     *
+     * The method must be called before {@link Javalin#start()}.
+     *
+     * @see <a href="https://javalin.io/documentation#static-files>Static files in docs</a>
+     */
     public Javalin enableStaticFiles(@NotNull String path, @NotNull Location location) {
         ensureActionIsPerformedBeforeServerStart("Enabling static files");
         staticFileConfig.add(new StaticFileConfig(path, location));
         return this;
     }
 
+    /**
+     * Context path (common prefix) for the instance.
+     */
     public String contextPath() {
         return this.contextPath;
     }
 
+    /**
+     * Sets the context path (common prefix) for the instance.
+     *
+     * The method must be called before {@link Javalin#start()}.
+     */
     public Javalin contextPath(@NotNull String contextPath) {
         ensureActionIsPerformedBeforeServerStart("Setting the context path");
         this.contextPath = Util.INSTANCE.normalizeContextPath(contextPath);
         return this;
     }
 
+    /**
+     * Port which is assigned to the instance.
+     */
     public int port() {
         return port;
     }
 
+    /**
+     * Sets the port to run the instance on.
+     *
+     * The method must be called before {@link Javalin#start()}.
+     */
     public Javalin port(int port) {
         ensureActionIsPerformedBeforeServerStart("Setting the port");
         this.port = port;
         return this;
     }
 
+    /**
+     * Sets internal logger level to {@link LogLevel#STANDARD}.
```

Comment:
Not really internal logger, this only affects the request-logging. "Sets request-logging-level" ?